app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsController.java

```diff
@@ -55,21 +60,34 @@ public void downloadFromService(boolean force) {
             // NOTE: the dummy data on the server contains a dummy ticket for event id 2
             //       thus, we add another event with id 2 for testing purposes
             // TODO: remove this when dummy data on server is made consistent or real data is used!
-            Event event = events.get(0);
+            /*Event event = events.get(0);
+
             events.add(new Event(2,event.getImage(), event.getTitle() + "2",
                     event.getDescription() + " Not to say extremely nice!",
                     event.getLocality(), event.getDate(), event.getLink()));
-
+            */
             eventDao.insert(events);
         } catch (IOException e) {
             Utils.log(e);
         }
 
         // Load all tickets
         try {
-            // TODO: replace by real user id (dummy ticket with user id 1 for now)
-            List tickets = api.getTickets(1);
-            ticketDao.insert(tickets);
+            if(Utils.getSetting(context, Const.CHAT_MEMBER, ChatMember.class) != null) {
+                api.getTickets(context, new Callback<List<Ticket>>() {
+                    @Override
+                    public void onResponse(Call<List<Ticket>> call, Response<List<Ticket>> response) {
+                        List<Ticket> list = response.body();
+                        if (list == null) list = new ArrayList<Ticket>();
+                        ticketDao.insert(list);
+                    }
+
+                    @Override
+                    public void onFailure(Call<List<Ticket>> call, Throwable t) {
+                        t.printStackTrace();
```

Comment:
Please add a `// TODO` here to do something useful. i.e. inform the user.

Reply:
I added alert controller messages where needed.